discord_bots/tasks.py

```diff
@@ -603,3 +610,39 @@ async def vote_passed_waitlist_task():
         ).delete()
         session.delete(vpw)
         session.commit()
+
+
+@tasks.loop(time=config.TRUESKILL_SIGMA_DECAY_JOB_SCHEDULED_TIME)
+async def apply_sigma_decay():
```

Comment:
Should be renamed to `sigma_decay_task` to match the style of the existing tasks